fuzz/src/test/java/com/dylibso/chicory/fuzz/SingleReproTest.java

```diff
@@ -32,7 +32,7 @@ void singleReproducer() throws Exception {
                 smith.run(seed.substring(0, Math.min(seed.length(), 32)), "test.wasm", types);
 
         var module = Module.builder(targetWasm).build();
-        var instance = module.instantiate(new HostImports(), false);
+        var instance = module.instantiate(new HostImports(), true, false);
```

Comment:
@andreaTP Not really important for this PR, but just a comment for later as we're looking at the a 1.0 API. I think booleans as params can get kind of confusing. I blame myself for starting this. Might make more sense to either have differently named instantiate functions or to have some kind of option builder type.

Reply:
Yes, completely agree, I'll take the time to run some cleanup at some point, and this is something that needs to improve.